src/dialog.jsx

```diff
@@ -143,9 +143,6 @@ function getStyles(props, state) {
       width: '100%',
       textAlign: 'right',
     },
-    paper: {
```

Comment:
Out of curiosity why was this removed? Maybe this was answered in the other PR already, sorry!